### ISSUE TITLE
Mention of "open-source" in README.md

### ISSUE THREAD

**Issue by cubuspl42** [facebook/facebook-android-sdk#1116]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [Github issues](https://github.com/facebook/facebook-android-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Java version

1.8

### Android version

API 29

### Android SDK version

15.0.1

### Installation platform & version

Gradle

### Package

Other / I don't know

### Goals

I wanted to read the README.md file to learn about Facebook Android SDK.

### Expected results

I wanted the content of the README.md file to be accurate and as close to being factually correct as possible. 

### Actual results

I didn't find the README.md file to be accurate and as close to being factually correct as possible, for the following reasons.

The description of the package starts with the following sentence: "This open-source library allows you to integrate Facebook into your Android app.".

This sentence may confuse many developers, because it suggests that this package is _open-source_.

At the same time, the license states:

> You are hereby granted a [...] license to use [...] and distribute this software [...] for use in connection with the web services and APIs provided by Facebook.

The dotted-out parts ([...]) do use liberal words like _worldwide_, _royalty-free_, _copy_, _modify_, and I excluded them not to imply that they are not effective, but to focus on what the sentence says, generally. Let's agree that the dotted-out parts let you do anything, limited only by what the rest of the sentence states.

What it does state is that you can use this software and its derivatives for one purpose **only**: to integrate Facebook into an Android app.

This doesn't meet the [Open Source Definition](https://opensource.org/osd) by Open Source Initiative. It definitely doesn't meet the [Free Software Definition](https://www.gnu.org/philosophy/free-sw.en.html) by the Free Software Foundation. It also doesn't meet the intuitive definition of open-source (in my opinion and in opinion of my software developer engineer friends I asked).

To sum up, it appears that this software doesn't meet any definition of open-source. Either the licence should be updated not to restrict use cases of this software, or the README.md file should be updated not to misinform its users and potential users.

### Steps to reproduce

Read the first sentence of README.md.

**Comment by LiquidLemon** [comment]
Oh wow, I never noticed this uses a proprietary license. Restricting the way this library is used definitely does not fit into what's generally understood as open source. Remember, "source available" is not the same thing as "open source".

This sentence makes it even worse:
> As with any software that integrates with the Facebook platform, your use of
this software is subject to the Facebook Platform Policy.

The license doesn't just apply this policy to software that integrates with Facebook using this SDK but also to any software that uses any part of this code for whatever reason.

Please stop misleading your users by making claims that this library is open source.

**Comment by jawwad** [comment]
Hi @cubuspl42 - Thanks for bringing up this inaccuracy to our attention. I've removed the word "open-source" from the README.
